experimental/GModule/GModule.jl

```diff
@@ -191,10 +196,27 @@ where `M` is a module over `R`.
 
 (modules over finite fields or number fields)
 """
-function descent_to_minimal_degree_field(M::GModule)
-  error("not yet ...")
+function descent_to_minimal_degree_field(C::GModule{<:Any, <:AbstractAlgebra.FPModule{fpFieldElem}})
+  return C
 end
 
+function descent_to_minimal_degree_field(C::GModule{<:Any, <:AbstractAlgebra.FPModule{<:FinFieldElem}})
+  #always over char field
+  K =  base_ring(C)
+  d = 0
+  while d < degree(K)-1
+    d += 1
+    degree(K) % d == 0 || continue
```

Comment:
would it make sense to iterate over `sort(divisors(degree(K)))` (and skip the last one) ? Ah well, probably doesn't matter for the degrees that we can actually use here.

Reply:
In our range, factoring is MUCH more expensive than trying